src/GuiBase/Utils/KeyMappingManager.cpp

```diff
@@ -319,47 +379,174 @@ KeyMappingManager::~KeyMappingManager() {
 
 std::string KeyMappingManager::enumNamesFromMouseButtons( const Qt::MouseButtons& buttons ) {
     std::string returnText;
+    std::string sep = "";
```

Comment:
Either prefer the initializer `{""}`or let the default constructor (empty string) to manage initialization. This prevent a copy...